instagram/src/components/CommentSection/CommentSection.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Comment from './Comment';
+import CommentInput from './CommentInput';
+import instaheart from './Images/instaheart.png';
+import comment from './Images/comment.png';
+
+import './CommentSection.css';
+
+class CommentSection extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      comments: props.comments,
+      newComment: '',
+      likes: 0
```

Comment:
It would be nice if likes was dynamically set to the starting value of dummyData